biz.aQute.bndlib/src/aQute/bnd/component/DSAnnotations.java

```diff
@@ -134,7 +146,7 @@ public boolean analyzeJar(Analyzer analyzer) throws Exception {
 
 		TreeSet<String> provides = new TreeSet<>();
 		TreeSet<String> requires = new TreeSet<>();
-		Version maxVersion = V1_0;
+		Version maxVersionUsedByAnyComponent = V1_0;
```

Comment:
This name grates on my sensibilities.  Perhaps maxComponentVersion? maxVersionEncountered? maxVersionUsed?

Reply:
I like `highestVersionUsed`.

Reply:
important to show that it is over all components and not per component